src/lib/core/CHIPConfig.h

```diff
@@ -1432,6 +1432,16 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_SCENES_MAX_EXTENSION_FIELDSET_SIZE_PER_CLUSTER 128
 #endif
 
+/**
+ * @brief The maximum number bytes taken by a scene. This needs to be increased if the number of clusters per scene is increased.
+ * @note The default number (256) is based on the assumption that the maximum number of clusters per scene is 3 and that those
+ * clusers are onOff, level control and color control cluster.
+ * @warning Changing this value will not only affect the RAM usage of a scene but also the size of the scene table in the flash.
+ */
+#ifndef CHIP_CONFIG_SCENES_MAX_SERIALIZED_SCENE_SIZE_BYTES
+#define CHIP_CONFIG_SCENES_MAX_SERIALIZED_SCENE_SIZE_BYTES 256
```

Comment:
If you could add in the comments the calculation that leads to 256 bytes that would be helpful in cases of increases.

3 clusters x (Max size for: OnOff( x Bytes), LevelControl(y Bytes), Color (z Bytes ) + Max NameLength 16 Bytes ) <= 256.

Something like that. With the real values for those clusters